src/storage/src/hummock/compactor/iterator.rs

```diff
@@ -320,6 +351,9 @@ impl ConcatSstableIterator {
             if start_index >= end_index {
                 found = false;
             } else {
+                self.task_progress
+                    .num_pending_read_io
+                    .fetch_add(1, std::sync::atomic::Ordering::SeqCst);
```

Comment:
We may miss to fetch_sub `num_pending_read_io` if `sstable_store.get_stream` failed but I guess that is fine because task progress will be cleared very soon in this case.

Reply:
We only record it in log when the task is cancel by meta-node